modules/oe_content_event/config/post_updates/20002_update_form_display/core.entity_form_display.node.oe_event.default.yml

```diff
@@ -3,7 +3,9 @@ status: true
 dependencies:
```

Comment:
Not sure if we should update the form display of the node, sites using this component might have customized form displays and this will mess with their configuration.

Reply:
Policy was changed, rmeoving